ScoutSuite/providers/base/authentication_strategy_factory.py

```diff
@@ -9,6 +9,9 @@
 }
 
 
+class InvalidAuthenticationStrategyException(Exception): pass
```

Comment:
Why this?

Reply:
Don't worry, it's just a bad merge, I'm fixing this.